crates/ruff_python_ast/src/whitespace.rs

```diff
@@ -17,41 +19,9 @@ pub fn indentation_at_offset<'a>(locator: &'a Locator, offset: TextSize) -> Opti
     let line_start = locator.line_start(offset);
     let indentation = &locator.contents()[TextRange::new(line_start, offset)];
 
-    if indentation.chars().all(char::is_whitespace) {
+    if indentation.chars().all(is_python_whitespace) {
         Some(indentation)
     } else {
         None
     }
 }
-
-/// Extract the leading words from a line of text.
-pub fn leading_words(line: &str) -> &str {
```

Comment:
These were moved, since they deal specifically with docstrings (and so the set of whitespace characters they respect differs from code).